zipkin-ui/js/component_ui/trace.js

```diff
@@ -443,7 +448,11 @@ export default component(function trace() {
     this.on(document, 'uiZoomInSpans', this.zoomInSpans);
     this.on(document, 'uiZoomOutSpans', this.zoomOutSpans);
 
+    this.on(document, 'uiShowJson', this.showJson);
+
     const self = this;
+    self.raw = data.raw;
```

Comment:
This PR does not provide a mechanism to do that. I suspect that is best saved for future work -- if there isn't a way to show an uploaded trace, no sense downloading the trace? 

I could be convinced otherwise, it would be easy to add an actual raw download link somewhere.